units/systemd-resolved.service.m4.in

```diff
@@ -25,8 +25,12 @@ Type=notify
 Restart=always
 RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-resolved
+User=systemd-resolve
+Group=systemd-resolve
+UMask=0022
 WatchdogSec=3min
-CapabilityBoundingSet=CAP_SETUID CAP_SETGID CAP_SETPCAP CAP_CHOWN CAP_DAC_OVERRIDE CAP_FOWNER CAP_NET_RAW CAP_NET_BIND_SERVICE
+CapabilityBoundingSet=CAP_SETPCAP CAP_NET_RAW CAP_NET_BIND_SERVICE
```

Comment:
Yes, you still need to drop the capabilities from the bounding set in all cases.